html/includes/graphs/application/ups-nut_voltage_battery.inc.php

```diff
@@ -48,7 +51,7 @@
         $i++;
     }
 } else {
-    echo "file missing: $rrd_filename";
+    echo "file missing: ".$rrd_filename;
```

Comment:
i think that almost graphs have it, just to display file missing when clicking show rrd command